instagram/src/App.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from "react";
+import "./App.css";
+import SearchBar from "./components/SearchBar/SearchBar";
+import PostContainer from './components/PostContainer/PostContainer';
+import dummyData from './dummy-data';
+ 
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      instaData : dummyData
+    };
+  }
+  render() {
+    return (
+      <div className="App">
+      <SearchBar />
```

Comment:
This is great but I would probably indent your `search bar` and `post container` components to make the nesting clearer.